README.md

```diff
@@ -7,7 +7,8 @@ This toolset runs TPC-H and TPC-DS benchmarks.
 
```

Comment:
First line of the readme should include a reference to SSB as well, or no listing of the benchmarks provided at all

Reply:
Done. Maybe at some point we can rename the whole thing to "s64-benchmark-toolkit" or similar.